integration-tests/cli/resource-deleter.it.js

```diff
@@ -100,467 +133,44 @@ describe('Resource Deleter', () => {
-    let stdout
-    let stderr
-    const resource = 'shippingMethods'
-
-    beforeAll(async () => {
-      filePath = tmp.fileSync().name
-      ;[stdout, stderr] = await exec(
-        `${bin} -o ${filePath} -p ${projectKey} -r ${resource}`
-      )
-    })
-
-    it('should log success messages', async () => {
-      expect(stderr).toBeFalsy()
-      expect(stdout)
-        .toMatch(/Starting to delete fetched shippingMethods/)
-        .toMatch(/All deleted/)
-    })
-
-    it('should delete shippingMethods', async () => {
-      const data = await fs.readFile(filePath, { encoding: 'utf8' })
-      expect(data)
-        .toBeDefined()
-        .toEqual('')
-        .toHaveLength(0)
-    })
-  })
-
-  describe('when deleting taxCategories', () => {
-    let filePath
-    let stdout
-    let stderr
-    const resource = 'taxCategories'
-
-    beforeAll(async () => {
-      filePath = tmp.fileSync().name
-      ;[stdout, stderr] = await exec(
-        `${bin} -o ${filePath} -p ${projectKey} -r ${resource}`
-      )
-    })
-
-    it('should log success messages', async () => {
-      expect(stderr).toBeFalsy()
-      expect(stdout)
-        .toMatch(/Starting to delete fetched taxCategories/)
-        .toMatch(/All deleted/)
-    })
-
-    it('should delete taxCategories', async () => {
-      const data = await fs.readFile(filePath, { encoding: 'utf8' })
-      expect(data)
-        .toBeDefined()
-        .toEqual('')
-        .toHaveLength(0)
-    })
-  })
-
-  describe('when deleting types', () => {
-    let filePath
-    let stdout
-    let stderr
-    const resource = 'types'
-
-    beforeAll(async () => {
-      filePath = tmp.fileSync().name
-      ;[stdout, stderr] = await exec(
-        `${bin} -o ${filePath} -p ${projectKey} -r ${resource}`
-      )
-    })
-
-    it('should log success messages', async () => {
-      expect(stderr).toBeFalsy()
-      expect(stdout)
-        .toMatch(/Starting to delete fetched types/)
-        .toMatch(/All deleted/)
-    })
-
-    it('should delete types', async () => {
-      const data = await fs.readFile(filePath, { encoding: 'utf8' })
-      expect(data)
-        .toBeDefined()
-        .toEqual('')
-        .toHaveLength(0)
-    })
-  })
-
-  describe('when deleting zones', () => {
-    let filePath
-    let stdout
-    let stderr
-    const resource = 'zones'
-
-    beforeAll(async () => {
-      filePath = tmp.fileSync().name
-      ;[stdout, stderr] = await exec(
-        `${bin} -o ${filePath} -p ${projectKey} -r ${resource}`
-      )
-    })
-
-    it('should log success messages', async () => {
-      expect(stderr).toBeFalsy()
-      expect(stdout)
-        .toMatch(/Starting to delete fetched zones/)
-        .toMatch(/All deleted/)
-    })
-
-    it('should delete zones', async () => {
-      const data = await fs.readFile(filePath, { encoding: 'utf8' })
-      expect(data)
-        .toBeDefined()
-        .toEqual('')
-        .toHaveLength(0)
+  describe.each([
+    'carts',
```

Comment:
sorted alphabetically would be appreciated

Reply:
Some resources are dependant on others (eg: you can't remove shipping info when there are any orders using it) so when sorted alphabetically it may throw some errors.